src/main/java/Duke.java

```diff
@@ -1,10 +1,319 @@
-public class Duke {
-    public static void main(String[] args) {
+import e0148811.duke.Deadline;
+import e0148811.duke.DukeException;
+import e0148811.duke.Event;
+import e0148811.duke.Task;
+import e0148811.duke.Todo;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Scanner;
+
+class Duke {
+    // a common error (invalid format) handling message prefix
+    static final String FORMAT_CORRECTION = "Invalid format for the instruction you gave.\n"
+            + "The correct format should be ";
+
+    public static void main(String[] args) throws IOException {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(logo);
+        Scanner sc = new Scanner(System.in);
+        List<Task> list = new ArrayList<>();
+        String instruction;
+
+        try {
+            BufferedReader br = new BufferedReader(new FileReader("data/duke.txt"));
+            boolean isEmptyFile = true;
+            String line;
+            while ((line = br.readLine()) != null) {
+                isEmptyFile = false;
+                addEventToList(list, line);
+            }
+            if (isEmptyFile) {
+                System.out.println("File found but empty. Start with an empty task list.");
+            } else {
+                System.out.println("File found. Load saved task list.");
+            }
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found. Start with an empty task list.");
+        }
+
+        System.out.println("Hello, this is Duke. "
+                + "Please give me an instruction followed by relevant description.");
+
+        while (!(instruction = sc.nextLine()).equals("bye")) {
```

Comment:
Instruction handling should be done on a separate class to make your code more readable and your logic easier to follow